code_generator/GdkCodeGenerator/Templates/UnityCommandPayloadGenerator.tt

```diff
@@ -44,6 +45,17 @@ namespace <#= qualifiedNamespace #>
 
                 public void Send<#= commandDetails.CommandName #>Response(<#= commandDetails.ResponseType #> response)
                 {
+                    if (Translation.<#= commandDetails.CommandName #>ResponsesToSend.Contains(RequestId))
```

Comment:
This approach here would detect if we attempt to send duplicate command responses during the same frame but it wouldn't be able to detect duplicate command responses after the first response was sent, can it? I.e. send a response in frame 1, send a duplicate response in frame 2.

Reply:
Presumably the components that allow sending responses get cleaned up at the end of the frame they're responded on? So the user would have to jump through some hoops to keep a reference around for a frame in order to hit this issue?

Reply:
Yup they won't get injected again but we already have the playerlifecycle system which stores it for later so I dont think it will be uncommon.